Gruntfile.js

```diff
@@ -0,0 +1,78 @@
+var fs = require('fs');
+var allProfiles;
+
+try {
+    allProfiles = require('./profiles.json');
+} catch (e) {
+    throw new Error('no profiles.json found. Please copy and customize profiles.json.dist');
```

Comment:
Please move this block into a grunt error, so the integration with other tools will be more straight forward 
```js
module.exports = function(grunt) {
    var profiles;
   ...
   } catch(err){
        grunt.fail.fatal('no profiles.json found. Please copy and customize profiles.json.dist');
    }
}
```